crates/sui-core/src/authority/consensus_quarantine.rs

```diff
@@ -375,7 +377,64 @@ impl ConsensusEphemeralOutput {
             shared_version_assignments: shared_version_assignments.into_iter().collect(),
             deferred_transactions: Mutex::new(deferred_transactions),
             user_signatures_for_checkpoints: Mutex::new(user_signatures_for_checkpoints),
+            metrics,
+        }
+    }
+
+    pub fn num_shared_version_assignments(&self) -> usize {
+        self.shared_version_assignments.len()
+    }
+
+    pub fn get_assigned_shared_object_versions(
+        &self,
+        key: &TransactionKey,
+    ) -> Option<Vec<(ConsensusObjectSequenceKey, SequenceNumber)>> {
+        self.shared_version_assignments
+            .get(key)
+            .map(|locks| locks.clone())
+    }
+
+    pub fn insert_shared_object_assignments(&self, versions: &AssignedTxAndVersions) {
+        debug!("set_assigned_shared_object_versions: {:?}", versions);
```

Comment:
was this meant to be left in? if so it doesn't seem to match the fn name?

Reply:
updated the name and changed to trace